process_jartic_archives.py

```
@@ -0,0 +1,281 @@
+                except Exception as e:
+                    pass  # Skip sensor errors
+    
+    # Yield remaining measurements
+    if measurements:
+        yield archive_index, month_str, measurements, location_count, len(locations)
+
+
+def write_measurements_batch(output_path: Path, measurements: list, append: bool = True):
+    """Write measurements to CSV file"""
+    mode = 'a' if append else 'w'
+    with open(output_path, mode) as f:
```

Pylint (3.3.7)

[warning] Using open without explicitly specifying an encoding

(W1514)